src/redux/actions.js

```diff
@@ -37,6 +41,9 @@ export function changeCamera(cameraPosition) {
 export function moveCamera(cameraPosition) {
   return { type: MOVE_CAMERA, data: cameraPosition };
 }
+export function movePointer(pointerData) {
```

Comment:
Looks like nothing is using this action creator or action type, so we can probably get rid of it